zokrates_js/README.md

```diff
@@ -6,34 +6,25 @@ JavaScript bindings for [ZoKrates](https://github.com/Zokrates/ZoKrates) project
 npm install zokrates-js
```

Comment:
Wouldn't it be better to link to the book from the readme? It would avoid duplication at the cost of an extra redirection in npm, I feel this is fine?

Reply:
yeah, seems like a reasonable thing to do here